packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderFinal/en.ftl

```diff
@@ -1,4 +1,5 @@
 verificationReminderFinal-subject = Final reminder to confirm your account
 verificationReminderFinal-description = A couple of weeks ago you created a { -product-firefox-account }, but never confirmed it. For your security, we will delete the account if not verified in the next 24 hours.
+verificationReminderFinal-description-2 = A couple of weeks ago you created a { -product-mozilla-account }, but never confirmed it. For your security, we will delete the account if not verified in the next 24 hours.
```

Comment:
I think we should deal with potential confusion here, but I can definitely seeing this being confusing in the couple of weeks following our rollout.

Reply:
Good point! Should we consider adding logic so that the Mozilla-branded email includes the string with `Firefox accounts` if the account was created before the rollout date?